development.ini

```diff
@@ -5,10 +5,7 @@ pyramid.includes =
     pyramid_debugtoolbar
     pyramid_tm
 
-SQLITE_DATABASE_URL = sqlite:///%(here)s/sqltutorial.sqlite
-POSTGRES_DATABASE_URL = postgresql+psycopg2://postgres:localpassword@localhost:5432/noderegistry
-sqlalchemy.url = %(POSTGRES_DATABASE_URL)s
-
+sqlalchemy.url = sqlite:///%(here)s/network_manager.sqlite
```

Comment:
I changed the default database name to network_manager.sqlite because sqltutorial.sqlite is from the tutorial and does not make sense as part of this project